partcad/src/partcad/project_config.py

```
@@ -25,7 +25,11 @@ class Configuration:
     name: str
 
     def __init__(
-        self, name, config_path=DEFAULT_CONFIG_FILENAME, include_paths=[]
+        self,
+        name: str,
+        config_path: str = DEFAULT_CONFIG_FILENAME,
+        include_paths: list[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)